_includes/markdown/JavaScript.md

```diff
@@ -103,7 +103,23 @@ Certain codebases may already have such libraries in place. Many legacy projects
 ### Concatenating Requests
 When constructing a page that contains a lot of client-side data requests you will want to consider concatenating your requests into a single Ajax call. This will help you avoid piling up requests or sending them through callbacks and nested promises when parts of the data depend on other parts.
 
-[GraphQL](https://graphql.org/) is an open source query language that can help with this situation by allowing you to combine multiple API requests into a single call. In a WordPress environment, the [WPGraphQL](https://github.com/wp-graphql/wp-graphql) plugin will let you directly access the WordPress JSON API.
+#### GraphQL
+[GraphQL](https://graphql.org/) is an open source data query and manipulation language. It provides an alternative to REST because it allows for a consistent way to make declarative queries. We first define the data structure(s) we need, then request the data, and return only the data that was requested. This creates an environment of smaller, more targeted calls to an API. It also allows us to concatenate multiple calls into single data requests, reducing the overhead and time to load.
+
+An essential part of a GraphQL API is an API schema. GraphQL requires a human-readable schema which describes the types which are available, and how they relate to one another. While writing a schema is reasonably straightforward, utilizing the standardized nature of WordPress’s database could save time. It may well be possible to reuse a schema from other projects, such as generated by a plugin like, [WPGraphQL](https://github.com/wp-graphql/wp-graphql).
+
+#### WPGraphQL
+If you are choosing to use GraphQL on a Wordpress project, it is recommended to use the WPGraphQL plugin. This plugin will make all the data in the WordPress JSON API available through a GraphQL endpoint - in many cases you won’t need to write the schema yourself. This will give you all the benefits of concatenating your data requests as well as easy access to your data as it is output by WordPress. You can retrieve your data by passing a query directly into a simple fetch request:
```

Comment:
Hey! Love seeing WPGraphQL in the best practices 😄 

> will make all the data in the WordPress JSON API available through a GraphQL endpoint

Not sure if the "WordPress JSON API" is referring to the WP REST API? If so, this is inaccurate as WPGraphQL does not wrap the WP REST API, it uses its own functions to resolve data and return a response. It _does_ use JSON to return data, per the GraphQL spec, but I don't think I'd refer to `wp_json_encode` as "the WordPress JSON API" 😉